modules/memberships/class-jetpack-memberships.php

```diff
@@ -208,17 +208,18 @@ public function render_button( $attrs ) {
 			'powered_text' => __( 'Powered by WordPress.com', 'jetpack' ),
 		);
 
-		$classes = array(
-			'wp-block-button__link',
-			'components-button',
-			'is-primary',
-			'is-button',
-			'wp-block-jetpack-' . self::$button_block_name,
-			self::$css_classname_prefix . '-' . $data['id'],
+		$classes = Jetpack_Gutenberg::block_classes(
+			self::$button_block_name,
+			$attrs,
+			array(
+				'wp-block-button__link',
+				'components-button',
+				'is-primary',
+				'is-button',
```

Comment:
Not entirely sure classes like this belong into a top-level block element, that's pre-existing. (It might raise the question if the new util is really needed, or more of a symptom of suboptimal architectural choices, but oh well.)